components/projectsView/ProjectsList.js

```diff
@@ -121,7 +121,7 @@ export default function ProjectsList(props) {
       </Row>
       {!showChartView && (
         <Row>
-          <Col className="text-right text-muted">
+          <Col className="text-end text-muted">
```

Comment:
every instance of `right` and `left` needs to be changed to `end` and `start`, respectively